scripts/extract-hvd-content.ts

```diff
@@ -35,16 +41,18 @@ const BASE_REPO_CONFIG = {
  * GitHub organization into the local filesystem that `dev-portal` can access.
  */
 ;(async function extractHvdContent() {
+	const isHvdRepoPreview =
+		process.env.PREVIEW_FROM_REPO === BASE_REPO_CONFIG.repo
 	// Skip extraction in deploy previews
-	if (isDeployPreview()) {
+	if (!isHvdRepoPreview && isDeployPreview()) {
 		console.log(
-			'Note: Content repo deploy preview detected. Skipping HVD content.'
+			'Note: Docs content repo deploy preview detected. Skipping HVD content.'
 		)
 		return
```

Comment:
If we do want to do the local data loading support, we may indeed want to skip over this and reference the local filesystem in the hvd-repo.